keras_nlp/models/bert.py

```diff
@@ -25,6 +25,27 @@ def _bert_kernel_initializer(stddev=0.02):
     return keras.initializers.TruncatedNormal(stddev=stddev)
 
 
+# Pretrained models
+BASE_PATH = "https://storage.googleapis.com/keras-nlp/models/"
+
+checkpoints = {
+    "bert_base": {
+        "bert_base_uncased": {
+            "md5": "074304b9d7f031ad5a6b626745f2a687",
+            "description": "Base size of Bert where all input is lowercased. "
+            "Trained on English wikipedia + books corpora.",
+            "vocabulary_size": 30522,
+        },
+        "bert_base_cased": {
+            "md5": "f94a6cb012e18f4fb8ec92abb91864e9",
+            "description": "Base size of Bert where case is maintained. "
```

Comment:
remove trailing space

Reply:
The whitespace was to separate the two lines of the description.